cime_config/tests.py

```diff
@@ -461,21 +461,21 @@
     "e3sm_scream_v1_lowres" : {
         "time"  : "01:00:00",
         "tests" : (
-            "ERP_D_Ln9.ne4_ne4.F2010-SCREAMv1",
+            "ERP_D_Ln9.ne4_oQU240.F2010-SCREAMv1.atmlndactive-rtm_off",
             "ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1",
-            "SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero",
-            "ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1",
-            "ERS_D_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1",
+            "SMS_D_Ln9.ne4_oQU240.F2010-SCREAMv1-noAero",
```

Comment:
I think it is good to have at least one test that still exercises MOSART. I think the SMS noAero test above does that, right?

Reply:
Yes, SMS test will do that (as it is not using the rtm_off testmod).